### PR TITLE
Fix signatures of call, check_call and check_output in subprocess

### PR DESCRIPTION
**Pull request by alvarocaceres**

### CODE REVIEW

File: stdlib/3/subprocess.pyi [hunk 1]
```diff
@@ -4,21 +4,61 @@
 
 from typing import Sequence, Any, Mapping, Callable, Tuple, IO, Optional, Union
 
-# TODO force keyword arguments
-# TODO more keyword arguments
-def call(args: Union[str, Sequence[str]], *, stdin: Any = ..., stdout: Any = ...,
-         stderr: Any = ..., shell: bool = ...,
+# Same args as Popen.__init__
+def call(args: Union[str, Sequence[str]],
```
Comment (matthiaskramm) [inline stdlib/3/subprocess.pyi]: This new definition drops the fact that all arguments except for args are keyword-only.
(That's the right thing to do for 2.7/, but I'm guessing that for 3/ the '*' was actually justified)

Comment (alvarocaceres) [inline stdlib/3/subprocess.pyi]: ah, didn't know about the new keyword-only-args feature -- but looks like the Popen in python3 isn't using it

Comment (alvarocaceres) [inline stdlib/3/subprocess.pyi]: ptal ?

File: stdlib/3/subprocess.pyi [hunk 2]
```diff
@@ -4,21 +4,61 @@
...
+                 shell: bool = ...,
+                 cwd: str = ...,
                  env: Mapping[str, str] = ...,
-                 cwd: str = ...) -> Any: ...
+                 universal_newlines: bool = ...,
+                 startupinfo: Any = ...,
+                 creationflags: int = ...,
+                 restore_signals: bool = ...,
+                 start_new_session: bool = ...,
+                 pass_fds: Any = ...) -> bytes: ...
```
Comment (sharmaeklavya2) [inline stdlib/3/subprocess.pyi]: This shouldn't be `bytes` because `check_output` returns `str` when `universal_newlines=True`.

Comment (alvarocaceres) [inline stdlib/3/subprocess.pyi]: ah, my bad, sorry -- will check for similar errors and send fix, unless you already have one

Comment (sharmaeklavya2) [inline stdlib/3/subprocess.pyi]: I have a fix.

Comment (sharmaeklavya2) [inline stdlib/3/subprocess.pyi]: Here's it - #348.

**Pull request merged**